keepassxc-browser/_locales/en/messages.json

```diff
@@ -914,6 +918,10 @@
         "message": "Let KeePassXC-Browser automatically fill in credentials if it receives only a single entry.",
         "description": "Auto-Fill Single Entry option help text."
     },
+    "optionsAutoFillRelevantCredentialHelpText": {
+        "message": "Let KeePassXC-Browser automatically fill in credentials if it receives relevant credential.",
```

Comment:
I think a better wording for this is: "Automatically fill in the relevant credential if a second fill is needed."

Reply:
fixed in [49c25f7](https://github.com/keepassxreboot/keepassxc-browser/pull/2409/commits/49c25f7711e15cca63a4ced3a8127ff80fae4875)